presses-universitaires-de-rennes-archeologie-et-culture.csl

```diff
@@ -414,14 +414,10 @@
       <key macro="title"/>
     </sort>
     <layout prefix="(" suffix=")" delimiter="&#160;; ">
-      <group delimiter="&#160;: ">
-        <group delimiter="&#160;">
-          <text macro="author-citation"/>
-          <text macro="year-date"/>
-        </group>
-        <text variable="locator"/>
-      </group>
-    </layout>
+     <text macro="author-citation" suffix=", "/>
+     <text macro="year-date" suffix=",&#160;p.&#160;"/>
+     <text variable="locator"/>
```

Comment:
@FrancoisC37, does the "p." stand for "page" here? If so, it doesn't belong on the "year-date" macro call. Instead, you could replace `<text variable="locator"/>` by:

```xml
<group delimiter="&#160;">
  <label variable="locator" form="short"/>
  <text variable="locator"/>
</group>
```

(and reduce the suffix on "year-date" to `suffix=",&#160;"`)